lib/spack/spack/cmd/info.py

```diff
@@ -17,6 +18,7 @@
 import spack.fetch_strategy as fs
 import spack.repo
 import spack.spec
+from spack.cmd.test import has_test_method
```

Comment:
If this function is used in multiple command, it might be better to move it in a core module (i.e. a module not starting with `spack.cmd`) and have both commands import that module.